### PR TITLE
Dev  tests2

### PR DESCRIPTION
**Pull request by samioan**
Refactoring handcheck function, game and log epics.

### CODE REVIEW

File: src/models/game/epics.js [hunk 1]
```diff
@@ -26,84 +25,94 @@ import {
...
     })
   );
 
+const startGameEpic = (action$, state$) =>
+  action$.pipe(ofType(betsPlaced.type), map(gameStarted));
+
+const foldEpic = (action$, state$) =>
+  action$.pipe(
+    ofType(fold.type),
+    map(() => {
```
Comment (stacon) [inline src/models/game/epics.js]: Short return the action here

File: src/models/game/reducer.js [hunk 2]
```diff
@@ -26,60 +27,75 @@ const initialState = {
 const gameReducer = (state = initialState, action) => {
   switch (action.type) {
     case gameStarted.type: {
+      return {
+        ...state,
+        phase: (state.phase = 1),
```
Comment (stacon) [inline src/models/game/reducer.js]: Typo?

File: src/models/game/reducer.js [hunk 3]
```diff
@@ -26,60 +27,75 @@ const initialState = {
...
         ...state,
         ...action.payload,
       };
     }
 
-    case cardReplaced.type: {
+    case playerFolded.type: {
+      return {
+        ...state,
+        phase: (state.phase = 4),
```
Comment (stacon) [inline src/models/game/reducer.js]: Typo?

**Review by stacon — CHANGES_REQUESTED** [review]
Check comments

File: src/models/game/epics.js (hunk 1 above)
Comment (samioan) [reply src/models/game/epics.js]: Done.

File: src/models/game/reducer.js (hunk 2 above)
Comment (samioan) [reply src/models/game/reducer.js]: Thought I had to reference state.phase, Fixed.

File: src/models/game/reducer.js (hunk 3 above)
Comment (samioan) [reply src/models/game/reducer.js]: Thought I had to reference state.phase, Fixed.

**Review by stacon — APPROVED** [review]

**Pull request merged**